src/components/calcite-chip/calcite-chip.scss

```diff
@@ -98,21 +100,26 @@
 }
 
 :host button {
-  @apply m-0 p-0 inline-flex self-stretch items-center border-none cursor-pointer;
+  @apply m-0 
+    p-0 
+    inline-flex 
+    self-stretch 
+    items-center 
+    border-none 
+    cursor-pointer
+    bg-transparent
+    duration-150
+    ease-in-out
+    text-color-1;
   -webkit-appearance: none;
-  background-color: transparent;
   border-radius: var(--calcite-chip-button-border-radius);
-  transition: $transition;
```

Comment:
fyi this transition variable from calcite-base is in our tailwind config as `@apply transition-default`